jscomp/others/node_buffer.ml

```diff
@@ -38,5 +38,8 @@ external fromStringWithEncoding : string -> ([ `ascii  | `utf8  | `utf16le  | `u
 external toString : t -> string = "toString"
 [@@bs.send]
 
+external toStringWithEncoding : (t, [ `ascii  | `utf8  | `utf16le  | `usc2  | `base64  | `latin1 | `binary  | `hex ] [@bs.string]) -> string = "toString"
+[@@bs.send]
+
```

Comment:
it's `t -> xx -> xx ` syntax

Reply:
My bad, mixed some Reason in there. ;) Fixed it.